spec/views/rb_taskboards/show_spec.rb

```diff
@@ -200,18 +207,16 @@
         end
       end
     end
-
   end
 
   describe 'update tasks or impediments' do
-
     it 'allows edit and drag for all tasks with the right permissions' do
       allow(User).to receive(:current).and_return(user1)
       task
       impediment
       render
 
-      assert_select ".model.work_package.task" do |task|
+      assert_select '.model.work_package.task' do |task|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.